.github/workflows/main.yml

```diff
@@ -33,6 +33,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: devonfw-tutorials/tutorial-compiler
+          path: tutorial-compiler
```

Comment:
If you change the path to the tutorial compiler, the paths in the 'npm install' and 'run buildRun.sh' step must also be changed. I am also not sure if the 'publish' step will still work properly then. I think it would be easiest to leave the path as it is and change the paths in your js.